### DISCUSSION TITLE
create change based on template

### DISCUSSION THREAD

**Discussion by KezHalls** [Snow-Shell/servicenow-powershell#232] [Q&A] [answered]
Hi,
Great tool.  thanks.
does this module handle creating a new change based on a template from the standard changes catalog

Thanks
Kez

**Comment by KezHalls** [comment]
Is this a no???? it doesn't

**Comment by gdbarron** [comment]
If you are referring to the service catalog, the module does not currently support this. You can always submit an enhancement and provide all the details for it to be built, or submit a pull request 😀

**Comment by gdbarron** [comment, marked as answer]
@KezHalls `New-ServiceNowChangeRequest` has been updated in the main branch with this.  Once one other issue is resolved, I'll get it published to the gallery.
